daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -181,7 +181,7 @@ execIde (Telemetry telemetry) (Debug debug) = NS.withSocketsDo $ Managed.runMana
       then Logger.GCP.gcpLogger (>= Logger.Warning) loggerH
       else pure loggerH
 
-    opts <- liftIO defaultOptionsIO
+    opts <- liftIO $ defaultOptionsIO Nothing
```

Comment:
I think we need to use the version we get from the `--target` flag here.

Reply:
I fix this in a separate PR, thanks!